hw/drivers/sensors/dps368/include/dps368/dps368.h

```diff
@@ -0,0 +1,374 @@
+#define     IFX_DPS368_INTR_SOURCE_BOTH                 2
+
+#define     IFX_DPS368_INTR_STATUS_REG_ADDR             0x0A
+#define     IFX_DPS368_INTR_STATUS_REG_LEN              1
+
+
+#define     IFX_DPS368_INTR_DISABLE_ALL                (uint8_t)0b10001111
+
+
+
+/*=======================================================================
+
+MACROS INTERNAL DEFINITIONS
+
+========================================================================*/
+#define POW_2_23_MINUS_1	0x7FFFFF   //implies 2^23-1
+#define POW_2_24			0x1000000
+#define POW_2_15_MINUS_1	0x7FFF
+#define POW_2_16			0x10000
+#define POW_2_11_MINUS_1	0x7FF
+#define POW_2_12			0x1000
+#define POW_2_20			0x100000
+#define POW_2_19_MINUS_1	524287
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef unsigned char dps_cfg;
+
+/* Struct to hold calibration coefficients read from device*/
+typedef struct
+{
+	/* calibration registers */
+  int16_t 	C0;		// 12bit
+  int16_t 	C1;		// 12bit
+
+  int32_t	C00;	// 20bit
+  int32_t   C10;	// 20bit
+  int16_t 	C01;	// 16bit
+  int16_t	C11;	// 16bit
+  int16_t	C20;	// 16bit
+  int16_t	C21;	// 16bit
+  int16_t	C30;	// 16bit
+
+}dps3xx_cal_coeff_regs_s;
+
+/* enum for seeting/getting device operating mode*/
+typedef enum
+{
+  DPS3xx_MODE_IDLE                   =  0b00000000,
+  /*One shot pressure measurement*/
+  DPS3xx_MODE_COMMAND_PRESSURE       =  0b00000001,
+ /*One shot temperature measurement */
+  DPS3xx_MODE_COMMAND_TEMPERATURE    =  0b00000010,
+  /*Continuous pressure measurement */
+  DPS3xx_MODE_BACKGROUND_PRESSURE    =  0b00000101,
+  /*Continuous temperature measurement */
+  DPS3xx_MODE_BACKGROUND_TEMPERATURE =  0b00000110,
+  /*Continuous temperature and pressure measurement */
+  DPS3xx_MODE_BACKGROUND_ALL         =  0b00000111,
+
+}dps3xx_operating_modes_e;
+
+/* enum of scaling coefficients either Kp or Kt*/
+typedef enum
+{
+    OSR_SF_1   = 524288,
+    OSR_SF_2   = 1572864,
+    OSR_SF_4   = 3670016,
+    OSR_SF_8   = 7864320,
+    OSR_SF_16  = 253952,
+    OSR_SF_32  = 516096,
+    OSR_SF_64  = 1040384,
+    OSR_SF_128 = 2088960,
+
+} dps3xx_scaling_coeffs_e;
+
+/* enum of oversampling rates for pressure and temperature*/
+typedef enum
+{
+    OSR_1   = 0b00000000,
+    OSR_2   = 0b00000001,
+    OSR_4   = 0b00000010,
+    OSR_8   = 0b00000011,
+    OSR_16  = 0b00000100,
+    OSR_32  = 0b00000101,
+    OSR_64  = 0b00000110,
+    OSR_128 = 0b00000111,
+
+} dps3xx_osr_e;
+
+/* enum of measurement rates for pressure*/
+typedef enum
+{
+    ODR_1   = 0b00000000,
+	ODR_2   = 0b00010000,
+	ODR_4   = 0b00100000,
+	ODR_8   = 0b00110000,
+	ODR_16  = 0b01000000,
+	ODR_32  = 0b01010000,
+	ODR_64  = 0b01100000,
+	ODR_128 = 0b01110000,
+
+} dps3xx_odr_e;
+
+/* enum of oversampling and measurement rates*/
+typedef enum
+{
+    TMP_EXT_ASIC = 0x00,
+    TMP_EXT_MEMS = 0x80,
+}dps3xx_temperature_src_e;
+
+typedef enum {
+		/*Reset for first time with all initialization steps and default
```

Comment:
Tabs and spacing

Reply:
Coding style related issues are addressed in new commits and are already fixed now.